plugins/@grouparoo/hubspot/src/lib/export/options.ts

```diff
@@ -94,18 +104,23 @@ class OptionsHandler {
     const names = [];
     if (customObject?.properties) {
       for (const object of customObject.properties) {
-        if (object.name.startsWith("hs_")) {
+        if (
+          object.archived ||
```

Comment:
In the future, maybe a standardized `shouldShowProperty` that we us in both places.